cfme/v2v/migrations.py

```diff
@@ -0,0 +1,329 @@
+import attr
+
+from navmazing import NavigateToAttribute, NavigateToSibling
+from widgetastic.widget import View
+from widgetastic_patternfly import Text, TextInput, Button, BootstrapSelect
+from widgetastic_manageiq import InfraMappingTreeView, MultiSelectList
+
+from cfme.base.login import BaseLoggedInPage
+from cfme.modeling.base import BaseCollection, BaseEntity
+from cfme.utils.appliance.implementations.ui import navigator, CFMENavigateStep, navigate_to
+
+
+# Views
+
+
+class InfraMappingCommonButtonsSet1(View):
```

Comment:
**Required**

`Set1` and `Set2` are really ambiguous names, let's find something more descriptive and applicable to what these buttons are.

Maybe something like `InfraMappingFormControlButtons` and `InfraMappingWizardCommon` - the second view isn't just buttons, it includes the tree view.